app/assets/sass/modules/_summary-panel.scss

```diff
@@ -1,6 +1,6 @@
 .payment-summary {
   margin-top: govuk-spacing(6);
-  border: govuk-spacing(2) solid $govuk-border-colour;
+  border-top: 2px solid $govuk-brand-colour;
```

Comment:
I guess using `govuk-spacing()` offered some uniformity and means to globally adjust spacing consistently. But if it doesn't offer an actual spacing of 2px and that is what is required then 🤷‍♂